tests/mock/clitools.rs

```diff
@@ -686,7 +686,12 @@ fn build_mock_channel(
     }
 }
 
-fn build_mock_unavailable_channel(channel: &str, date: &str, version: &'static str) -> MockChannel {
+fn build_mock_unavailable_channel(
+    channel: &str,
+    date: &str,
+    version: &'static str,
```

Comment:
This `'static` is probably no longer needed.

Reply:
Indeed, I have pruned!